deployment/packer/template.js

```diff
@@ -0,0 +1,82 @@
+{
+    "variables": {
+        "version": "",
+        "branch": "",
+        "aws_region": "",
+        "aws_ubuntu_ami": "",
+        "stack_type": ""
+    },
+    "builders": [
+        {
+            "name": "mmw-app",
+            "type": "amazon-ebs",
+            "region": "{{user `aws_region`}}",
+            "source_ami": "{{user `aws_ubuntu_ami`}}",
+            "instance_type": "m3.large",
+            "ssh_username": "ubuntu",
+            "ami_name": "mmw-app-{{timestamp}}",
+            "run_tags": {
+                "PackerBuilder": "amazon-ebs"
+            },
+            "tags": {
+                "Name": "mmw-app",
+                "Version": "{{user `version`}}",
+                "Branch": "{{user `branch`}}",
+                "Created": "{{ isotime }}",
+                "Service": "Application",
+                "Environment": "{{user `stack_type`}}"
+            },
+            "associate_public_ip_address": true
+        },
+        {
+            "name": "mmw-worker",
+            "type": "amazon-ebs",
+            "region": "{{user `aws_region`}}",
+            "source_ami": "{{user `aws_ubuntu_ami`}}",
+            "instance_type": "m3.large",
+            "ssh_username": "ubuntu",
+            "ami_name": "mmw-app-{{timestamp}}",
```

Comment:
Fixed in https://github.com/WikiWatershed/model-my-watershed/commit/2d179f02e9f514fc69591ad2d01424ecd0e74aa7.